packages/angular-cli/models/webpack-build-development.ts

```diff
@@ -1,14 +1,3 @@
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-
 export const getWebpackDevConfigPartial = function(projectRoot: string, appConfig: any) {
-  return {
-    output: {
-      filename: '[name].bundle.js',
-      sourceMapFilename: '[name].bundle.map',
-      chunkFilename: '[id].chunk.js'
-    },
-    plugins: [
-      new ExtractTextPlugin({filename: '[name].bundle.css'})
-    ]
-  };
+  return { };
```

Comment:
Not sure.  But I'll see if I can put a PR together.
Also, I was looking through `webpack-config.ts` where the merging happens and to cleanly remove the dev config will require some refactoring (which the file could probably use either way) so it might be best to leave it for a followup PR.